tools/logger/convert.c

```diff
@@ -51,6 +52,29 @@ struct proc_ldc_entry {
 
 static const char *BAD_PTR_STR = "<bad uid ptr %x>";
 
+static void log_err(const char *fmt, ...)
+{
+	ssize_t needed_size;
+	va_list args;
+	char *buff;
+
+	va_start(args, fmt);
+	needed_size = vsnprintf(NULL, 0, fmt, args) + 1;
+	buff = malloc(needed_size);
+	if (buff) {
+		vsprintf(buff, fmt, args);
```

Comment:
Don't you need to va_end and va_start args again?

Reply:
`va_copy`  perhaps will be better, but yes there should be fix, thanks

Reply:
True that.